store/src/java/com/zimbra/cs/service/admin/ModifyCos.java

```diff
@@ -66,6 +69,26 @@ public Element handle(Element request, Map<String, Object> context) throws Servi
         return response;
     }
 
+    void validateMailAttachmentMaxSize(Map<String, Object> attrs) throws ServiceException {
```

Comment:
Can we create this as a common method that can be used both in the ModifyAccount and ModifyCos classes? There is a parent class AdminDocumentHandler, check if we can create the same there

While creating a method, better  specify an access modified rather than using default

In this case, any changes that happen in the future need to be considered in both places.

Reply:
Created the method in AdminDocumentHandler class.